packages/api/test/__fixtures__/sdk/operationid-quirks/index.ts

```diff
@@ -80,6 +80,23 @@ class SDK {
     return this.core.fetch('/quirky-operationId', 'get');
   }
 
+  /**
+   * This operation doesn't have an `operationId` so we should create one.
+   *
+   */
+  getNoOperationId(): Promise<FetchResponse<number, unknown>> {
```

Comment:
This wasn't being added due to the aforementioned bug with operations not having an `operationId` not being added into SDKs.